docs/reference-implementation/installations/idpbuilder/usage.md

```diff
@@ -190,7 +190,7 @@ curl -k -X POST \
 
 </details>
 
-### Custom Packages
+## Custom Packages
 
 Idpbuilder supports specifying custom packages using the flag `-p` flag. This flag expects a directory (local or remote) containing ArgoCD application files and / or ArgoCD application set files. In case of a remote directory, it must be a directory in a git repository, and the URL format must be a [kustomize remote URL format](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/remoteBuild.md).
```

Comment:
Should we also mention that we support too ApplicationSet ?

Reply:
Let's do that in another PR. This PR was about using resource hooks.